### PR TITLE
Updated dependencies

### PR DESCRIPTION
**Pull request by ulfarsson**
Update dependencies

### CODE REVIEW

File: comb_spec_searcher/strategies/constructor/cartesian.py [hunk 1]
```diff
@@ -334,12 +334,12 @@ def random_sample_sub_objects(
...
-        return tuple(
-            subsampler(n=extra_params.pop("n"), **extra_params)
-            for subsampler, extra_params in zip(subsamplers, extra_parameters)
-        )
+                assert extra_parameters is not None
+                return tuple(
+                    subsampler(n=extra_params.pop("n"), **extra_params)
+                    for subsampler, extra_params in zip(subsamplers, extra_parameters)
+                )
+        raise Exception("Function did not return")
```
Comment (enadeau) [inline comb_spec_searcher/strategies/constructor/cartesian.py:342]: It is bad practice to raise a plain exception. Maybe RuntimeError would be more appropriate here

File: comb_spec_searcher/strategies/constructor/disjoint.py [hunk 2]
```diff
@@ -174,19 +174,19 @@ def random_sample_sub_objects(
...
-            + (obj,)
-            + tuple(None for _ in range(len(subrecs) - idx - 1))
-        )
+                obj = subsampler(n=n, **extra_params)
+                return (
+                    tuple(None for _ in range(idx))
+                    + (obj,)
+                    + tuple(None for _ in range(len(subrecs) - idx - 1))
+                )
+        raise Exception("Function did not return")
```
Comment (enadeau) [inline comb_spec_searcher/strategies/constructor/disjoint.py:189]: Same comment as for cartesian

File: comb_spec_searcher/strategies/constructor/cartesian.py (hunk 1 above)
Comment (ulfarsson) [reply comb_spec_searcher/strategies/constructor/cartesian.py]: Fixed

File: comb_spec_searcher/strategies/constructor/disjoint.py (hunk 2 above)
Comment (ulfarsson) [reply comb_spec_searcher/strategies/constructor/disjoint.py]: Fixed

**Review by christianbean — APPROVED** [review]

**Review by enadeau — APPROVED** [review]

**Pull request merged**